app/assets/stylesheets/application.scss

```diff
@@ -120,7 +120,7 @@ a {
 }
 
 body {
-	background-color: #657786;
+	background-color: #14171a;
```

Comment:
if you're looking to achieve some consistency with colors, storing colors into scss variables would be helpful